crates/sui-core/src/epoch/randomness.rs

```diff
@@ -148,7 +149,7 @@ impl RandomnessManager {
                 .try_into()
                 .expect("validity threshold should fit in u16"),
             protocol_config.random_beacon_reduction_allowed_delta(),
-            protocol_config.random_beacon_reduction_lower_bound(),
+            protocol_config.random_beacon_reduction_lower_bound() as u16,
```

Comment:
i specifically chose type of this to match type of the argument in fastcrypto and avoid casts, is there a reason it now needs to change?

Reply:
everything in fastcrypto uses u16 to be consistent, but I didn't think we have to change this flag

Reply:
as discussed would be good if you could change this to `try_into().expect`